engine/src/main/java/org/hibernate/validator/HibernateValidatorConfiguration.java

```diff
@@ -62,6 +63,16 @@
 	String CONSTRAINT_MAPPING_CONTRIBUTORS = "hibernate.validator.constraint_mapping_contributors";
 
 	/**
+	 * Property for configuring script evaluator factory, allowing to set up which factory will be used to create
+	 * {@link org.hibernate.validator.cfg.scriptengine.ScriptEvaluator} for evaluation of script expressions in
```

Comment:
Let's use unqualified names (via imports) in JavaDocs for the sake of readability of rendered docs.